js/packages/common/src/contexts/meta/loadAccounts.ts

```diff
@@ -1155,6 +1155,7 @@ export const initMetadata = async (
     await metadata.info.init();
     setter('metadataByMint', metadata.info.mint, metadata);
     setter('metadata', '', metadata);
+    setter('metadataByMetadata', metadata.pubkey, metadata);
     const masterEditionKey = metadata.info?.masterEdition;
```

Comment:
This form of update state is only used in loadAccounts. Also setting the metadataByMetadata key added with storefront auction caching for easier lookup. List requires a find operation.